src/storage/exec/IndexEdgeScanNode.cpp

```diff
@@ -0,0 +1,147 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "storage/exec/IndexEdgeScanNode.h"
+namespace nebula {
+namespace storage {
+IndexEdgeScanNode::IndexEdgeScanNode(const IndexEdgeScanNode& node)
+    : IndexScanNode(node), edge_(node.edge_) {}
+IndexEdgeScanNode::IndexEdgeScanNode(RuntimeContext* context,
+                                     IndexID indexId,
+                                     const std::vector<cpp2::IndexColumnHint>& columnHint,
+                                     ::nebula::kvstore::KVStore* kvstore)
+    : IndexScanNode(context, "IndexEdgeScanNode", indexId, columnHint, kvstore) {
+  getIndex = std::function([this](std::shared_ptr<IndexItem>& index) {
```

Comment:
Just to facilitate unittest. In the unittest, we can assign different functions to `getIndex` as needed, instead of constructing complicated `storageEnv`. Perhaps a more reasonable way is to derive `IndexScanNode` and override the `getIndex` function, but this way is a bit cumbersome.

Reply:
I have read and check the unit test. I still prefer derive it, but ok to me.